Sources/ArcGISToolkit/Components/OfflineMapAreasView/PreplannedListItemView.swift

```diff
@@ -35,6 +38,13 @@ public struct PreplannedListItemView: View {
         .task {
             await model.load()
         }
+        .onReceive(model.$result) { result in
+            model.updateDownloadStatus(for: result)
+        }
+        .onChange(of: model.job?.status) { status in
+            guard canShowNotifications else { return }
+            model.notifyJobCompleted()
```

Comment:
I'm not sure this can be done on the model. The job is a published property but changes to its status cannot be observed with `didSet`. Do you have a different suggestion?

Reply:
The job status is streamed. That means it can be observed.

Reply:
Something like: 

```swift
for await status in foundJob.$status { ... }
```